test/flip.jl

```diff
@@ -15,4 +15,22 @@
         # neighboring nodes
         @test flip_prop.D₂ in neighbor_districts
     end
+
+    @testset "flip_chain()" begin
+        # this is a dummy constraint
+        pop_constraint = PopulationConstraint(graph, "population", 10.0)
+        cont_constraint = cont_constraint = ContiguityConstraint()
+        scores = ["electionD", "electionR", "purple", "pink"]
+        num_steps = 2 # test 2 steps for now

```

Comment:
how long does this take when it's bumped up (e.g. to 1k steps)?

Reply:
Great q! I ran the following code: `using Pkg; @time Pkg.test()`

With 2 steps for flip chain:  17.040888 seconds (658.31 k allocations: 39.720 MiB, 0.13% gc time)
With 1000 steps for flip chain: 15.998416 seconds (658.31 k allocations: 39.722 MiB)

Obviously this is pretty unintuitive, so I'm guessing it just means that the performance is minimal to nil? 

Also btw this helped me catch an error in `src/flip.jl` so thanks for that :D

Reply:
And just to be clear, I am bumping up the test to 1000 steps!